src/resolvers.ts

```diff
@@ -22,15 +22,25 @@ export const resolveConfigPath = async (path: Arrayable<string>) => (
  * @param srcDir
  * @returns array of resolved content globs
  */
-export const resolveContentPaths = (srcDir: string) => ([
-  `${srcDir}/components/**/*.{vue,js,ts}`,
-  `${srcDir}/layouts/**/*.vue`,
-  `${srcDir}/pages/**/*.vue`,
-  `${srcDir}/composables/**/*.{js,ts}`,
-  `${srcDir}/plugins/**/*.{js,ts}`,
-  `${srcDir}/utils/**/*.{js,ts}`,
-  `${srcDir}/{App,app}.{js,ts,vue}`,
-  `${srcDir}/{Error,error}.{js,ts,vue}`,
+export const resolveContentPaths = (srcDir: string, nuxt = useNuxt()) => ([
+  // `${srcDir}/components/**/*`,
```

Comment:
components directory seems extremely tricky

Reply:
nvm, used resolveAlias. Getting each component seems to be run-time scope rather than build-time.